test/CMakeLists.txt

```diff
@@ -53,13 +53,44 @@ string(RANDOM LENGTH 9 ALPHABET 0123456789 RANDOM_SEED)
 message(STATUS "Random seed is ${RANDOM_SEED}")
 set(TEST_PARAM_DELIM ":")
 
-list(APPEND LUATEST_COMMAND_WITH_FLAGS
-    ${PROJECT_SOURCE_DIR}/test-run/lib/luatest/bin/luatest
-    --no-clean
-    --verbose
-    --no-capture
-    --output tap
-)
+function(add_luatest_test suite_name test_name shuffle)
+    set(test_title "test/${suite_name}/${test_name}")
+    set(test_log "${VARDIR}/${suite_name}-${test_name}.log")
+    set(test_env
+        "LUA_PATH=${LUATEST_LUA_PATH}"
+        "LUA_CPATH=${LUA_CPATH}"
+        "TARANTOOL_SRC_DIR=${PROJECT_SOURCE_DIR}"
+        "SOURCEDIR=${PROJECT_SOURCE_DIR}"
+        "BUILDDIR=${PROJECT_BINARY_DIR}"
+        "PYTHON_EXECUTABLE=${PYTHON}"
+        "VARDIR=${VARDIR}"
+    )
```

Comment:
`test_env` is different for suites, please parameterize `add_luatest_test suite_name()` by test env as well.

Reply:
thanks!